components/admin/ApplicationsTable.tsx

```diff
@@ -0,0 +1,61 @@
+import { FC } from "react";
+
+type Student = {
+  id: number;
```

Comment:
we're using uuid's that are strings so would be better to make this type `string`

Reply:
Ok, what will the uuid be used for?